Formula/imagemagick.rb

```diff
@@ -72,49 +73,51 @@ def install
       --enable-static
     ]
 
-    if build.without? "modules"
-      args << "--without-modules"
-    else
-      args << "--with-modules"
-    end
-
-    if build.with? "opencl"
-      args << "--enable-opencl"
-    else
-      args << "--disable-opencl"
-    end
-
-    if build.with? "openmp"
-      args << "--enable-openmp"
-    else
-      args << "--disable-openmp"
-    end
-
-    if build.with? "webp"
-      args << "--with-webp=yes"
-    else
-      args << "--without-webp"
+    %W[
+      hdri hdri
+      opencl opencl
+      openmp openmp
+      zero-configuration zero-configuration
+    ].each_slice(2) do |option, feature|
+      if build.with? option
+        args << "--enable-#{feature}"
+      else
+        args << "--disable-#{feature}"
+      end
     end
 
-    if build.with? "openjpeg"
-      args << "--with-openjp2"
-    else
-      args << "--without-openjp2"
+    %W[
+      fftw fftw
```

Comment:
The enable/disable block ahead could certainly be reduced to one variable, but I don't see much benefit of refactoring here. But I don't feel strongly either way, so will do.

> also I prefer %w if there's no interpolation

To me that smells like a style violation (only in principle; in practice I don't think `audit` catches it). `%w` stands for single quotes whereas `%W` stands for double quotes, and apparently double quotes is the default quote throughout Homebrew's codebase.